espanol/calcetines/calcetines.md

```diff
@@ -1,2 +1,3 @@
 El Consejero ordena a Beatito que traigan los calcetines sucios de los gigantes para el caldo de la cena.
-Los hongos son nutritivos y pueden volver mas fuertes a los gerreros.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.